app/src/main/java/com/mr/nemo/dragonfly/domain/entity/auth/signup/PasswordValue.kt

```diff
@@ -0,0 +1,6 @@
+package com.mr.nemo.dragonfly.domain.entity.auth.signup
+
+data class PasswordValue(
+    val password: String = "",
+    val confirmation: String = ""
+)
```

Comment:
@bohdanzdaniekin is it a good idea to have `confirmation` here too?

Reply:
![image](https://github.com/bohdanzdaniekin/DragonFly/assets/23140030/54ef270a-2247-4ecd-a52c-f636067bfb38)
these values are tightly coupled, so I'm not sure I should separate them into different classes/properties.